### PR TITLE
Add section about configuring user authentication in Docker Compose

### PR DESCRIPTION
**Pull request by kgolubic**
### Description

- Added section about configuring user authentication in Docker Compose
- Added info on usage of  `MEMGRAPH_USER` and `MEMGRAPH_PASSWORD` env vars

### Pull request type

Please check what kind of PR this is:

- [ ] Fix or improvement of an existing page

### Related PRs and issues

PR this doc page is related to: 
(especially necessary if the PR is related to a release)

Closes:
- https://github.com/memgraph/documentation/issues/640
- https://github.com/memgraph/documentation/issues/332

### Checklist:

- [ ] Check all content with Grammarly
- [ ] Perform a self-review of my code
- [ ] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors
- [ ] Add a corresponding label
- [ ] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

File: pages/getting-started/install-memgraph/docker-compose.mdx [hunk 1]
```diff
@@ -167,6 +167,65 @@ the MAGE library included, you won't be able to use graph algorithms.
 > Want to see applications built with Memgraph and Docker Compose? Check out
 > [Memgraph's Github](https://github.com/memgraph) repositories.
 
+## Configuring user authentication in Docker Compose
+
+When setting up Memgraph with Docker Compose, you can use environment variables
+to manage user authentication. You can use `MEMGRAPH_USER` and
+`MEMGRAPH_PASSWORD` to either create a new user or connect with existing
+credentials.
```
Comment (matea16) [inline pages/getting-started/install-memgraph/docker-compose.mdx:175]: `MEMGRAPH_USER` and `MEMGRAPH_PASSWORD` env vars only create user if it doesn't already exist. When it comes to connection, you still have to manually enter the credentials when trying to access Memgraph. 

```suggestion
When setting up Memgraph with Docker Compose, you can use environment variables
 `MEMGRAPH_USER` and
`MEMGRAPH_PASSWORD` to create a new user if that username doesn't already exist.
```

**Comment by matea16** [comment]
I added one comment about the env vars. There are few scenarios that I tested out.
1. If the user doesn't already exist, it will be created after adding the MEMGRAPH_USER and MEMGRAPH_PASSWORD env vars. You still have to manually insert those credentials when accessing Memgraph Lab.
2. If the user with that username exists and you, for example, list the wrong password, nothing will happen. The user won't be created, the password will remain the same as originally set and you will have to manually insert the original credentials when accessing Memgraph Lab
3. If you have created a user in Lab and on the next startup of the docker-compose file you don't list the env vars, nothing will happen as well. You'll be able to connect to Memgraph normally, but you'll have to manually insert those credentials when accessing Memgraph Lab.

**Review by matea16 — CHANGES_REQUESTED** [review]

File: pages/configuration/configuration-settings.mdx [hunk 2]
```diff
@@ -198,8 +198,8 @@ This section contains the list of all other releant flags used within Memgraph.
...
-| MEMGRAPH_USER     | Username                                                                                                                                                                                                                  | `[string]` |
-| MEMGRAPH_PASSWORD | User password                                                                                                                                                                                                             | `[string]` |
+| MEMGRAPH_USER     | Specifies the username to connect to Memgraph. If the user does not already exist, Memgraph will create a new user with this username.                                                                                    | `[string]` |
+| MEMGRAPH_PASSWORD | Specifies the password for the user. If the user is being created (does not already exist), this password will be assigned. Otherwise, it is used to authenticate the connection to Memgraph.                             | `[string]` |
```
Comment (katarinasupe) [inline pages/configuration/configuration-settings.mdx:202]: Here, it applies the same that Matea said in other comments.

**Review by katarinasupe — CHANGES_REQUESTED** [review]

**Comment by kgolubic** [comment]
> I added one comment about the env vars. There are few scenarios that I tested out.
> 
> 1. If the user doesn't already exist, it will be created after adding the `MEMGRAPH_USER` and `MEMGRAPH_PASSWORD` env vars. You still have to manually insert those credentials when accessing Memgraph Lab.
> 2. If the user with that username exists and you, for example, list the wrong password, nothing will happen. The user won't be created, the password will remain the same as originally set and you will have to manually insert the original credentials when accessing Memgraph Lab
> 3. If you have created a user in Lab and on the next startup of the docker-compose file you don't list the env vars, nothing will happen as well. You'll be able to connect to Memgraph normally, but you'll have to manually insert those credentials when accessing Memgraph Lab.

Thank you @matea16! I'll add all of this info as additional Callout.

File: pages/configuration/configuration-settings.mdx (hunk 2 above)
Comment (kgolubic) [reply pages/configuration/configuration-settings.mdx]: I've updated the description of ENV variables.

**Pull request merged**